src/iap/shared/components/carousel/carousel.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="iap-carousel">
+	<div class="iap-carousel__slide-container">
+		<ng-template #slideHost></ng-template>
+	</div>
+
+	<div
+		class="iap-carousel__prev-button"
+		(click)="onPrevButtonClick()"
+	>
+		<span class="iap-carousel__prev-button-label"><</span>
```

Comment:
Typo :point_right: :point_left: 
```suggestion
		<span class="iap-carousel__prev-button-label"></span>
```